myia/unify.py

```diff
@@ -313,8 +327,12 @@ def unify_union(self, w: UnionVar[T], v: Value[T],
             equiv[diff] = UnionVar(set(okv[diff] for okv in ok_values))
             return equiv
 
-    def unify_raw(self, w: Value[T], v: Value[T],
-                  equiv: EquivT[T]) -> EquivT[T]:
+    def _getvar(self, v):
+        if hasattr(v, '__var__'):
+            return v.__var__()
```

Comment:
I know I had made it a method originally in #52, but I think it may be better to use a `__var__` attribute than a method.

Reply:
ok